src/components/Navigation/RootNavigatorExtraContent/index.native.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import TopLevelNavigationTabBar from '@components/Navigation/TopLevelNavigationTabBar';
+import type RootNavigatorExtraContentProps from './types';
+
+function RootNavigatorExtraContent({state}: RootNavigatorExtraContentProps) {
+    return <TopLevelNavigationTabBar state={state} />;
+}
```

Comment:
Let's explain why we don't render `SidePanel` there, like we do on other platforms